src/gui/Src/Gui/CPUStack.cpp

```diff
@@ -532,7 +532,14 @@ void CPUStack::mouseDoubleClickEvent(QMouseEvent* event)
 
     default:

```

Comment:
I think you should add `case 1: // value` and then call `modifySlot()` as it is now, that way none of the behavior disappears